client/src/components/Sub_ProductDetail/AddToCart.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import SizeSelector from './Sub_AddToCart/SizeSelector.jsx';
+import QuantitySelector from './Sub_AddToCart/QuantitySelector.jsx';
+
+function AddToCart({ skus, startSelect, currentStyle }) {
+  const [skuID, setSkuID] = useState();
+  const [quantity, setQuantity] = useState(1);
+  const [showAlert, setShowAlert] = useState(false);
+  const [validQuantities, setValidQuantities] = useState([1]);
+  const validSkus = Object.entries(skus).filter((entry) => entry[1].quantity > 0);
+
+  const handleSize = (e) => { setSkuID(e.target.value); setQuantity(1); };
```

Comment:
very clean 1 line event handler.

Reply:
learned this Object.entries from your code